examples/notebooks/beam-ml/run_inference_tensorflow_hub.ipynb

```diff
@@ -0,0 +1,307 @@
+      "metadata": {
+        "id": "74db0203-3d26-4bc4-8271-81fad9756297"
+      },
+      "source": [
+        "## Create a Keras Model from TensorFlow Hub Image"
+      ],
+      "id": "74db0203-3d26-4bc4-8271-81fad9756297"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "r1fgn5I_-mzA"
+      },
+      "outputs": [],
+      "source": [
+        "from google.colab import auth\n",
+        "auth.authenticate_user()"
+      ],
+      "id": "r1fgn5I_-mzA"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "2ff8e394-f577-4dea-bef9-a4f4528c1378"
+      },
+      "outputs": [],
+      "source": [
+        "PROJECT_ID = '<Project Id>'\n",
+        "GCS_BUCKET = '<GCS Bucket>'\n",
+        "\n",
+        "MODEL_PATH = f'{GCS_BUCKET}/st5-base/1'"
+      ],
+      "id": "2ff8e394-f577-4dea-bef9-a4f4528c1378"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ccaede25-1c1a-4ec4-9296-25c9a2ac43d7"
+      },
+      "outputs": [],
+      "source": [
+        "inp = tf.keras.layers.Input(shape=[], dtype=tf.string, name='input')\n",
+        "hub_url = \"https://tfhub.dev/google/sentence-t5/st5-base/1\"\n",
+        "imported = hub.KerasLayer(hub_url)\n",
+        "outp = imported(inp)\n",
+        "model = tf.keras.Model(inp, outp)\n",
+        "\n",
+        "# Sentenced-T5 model returns a 768-dimensional vector for an English text input.\n",
+        "# Note the 'input' that we will pass in as example's feature key name.\n",
+        "model.summary()"
+      ],
+      "id": "ccaede25-1c1a-4ec4-9296-25c9a2ac43d7"
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "29803d5b-93b9-41fc-b414-f7c737c5d7bc"
+      },
+      "source": [
+        "## Save the model with a TF function definition for RunInference()"
+      ],
+      "id": "29803d5b-93b9-41fc-b414-f7c737c5d7bc"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "704abfca-5956-4fc1-9717-4c6d5bf2db8e"
+      },
+      "outputs": [],
+      "source": [
+        "RAW_DATA_PREDICT_SPEC = {\n",
+        "    'input': tf.io.FixedLenFeature([], tf.string),\n",
+        "}\n",
+        "\n",
+        "@tf.function(input_signature=[tf.TensorSpec(shape=[None], dtype=tf.string)])\n",
+        "def call(serialized_examples):\n",
+        "    features = tf.io.parse_example(serialized_examples, RAW_DATA_PREDICT_SPEC)\n",
+        "    return model(features)\n",
+        "\n",
+        "tf.saved_model.save(model, MODEL_PATH, signatures={'serving_default': call})"
+      ],
+      "id": "704abfca-5956-4fc1-9717-4c6d5bf2db8e"
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7b56569d-e540-44ed-a46a-9cec886522f6"
+      },
+      "source": [
+        "## Create and test a RunInference pipeline locally"
```

Comment:
Where are we testing the output? the heading looks misleading if we are not actually testing the output

Reply:
I think this is actually somewhat clear if you look at how the notebook renders - this applies to all remaining sections - https://github.com/apache/beam/blob/3de2f85b3439a7084b60eb600a56bdc4cd36525c/examples/notebooks/beam-ml/run_inference_tensorflow_hub.ipynb

Reply:
Done. Changed `test` to `run`.